hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -201,7 +209,11 @@ def _bank_add_signals(
             abort(400, f"Invalid {name} signal: {str(e)}")
 
     content_config = iface.BankContentConfig(
-        id=0, disable_until_ts=0, collab_metadata={}, original_media_uri=None, bank=bank
```

Comment:
@Dcallies was this meant to add new content with `disable_until_ts=0` ?

actually it looks like `disable_until_ts` and `collab_metadata` don't get used in `storage.bank_add_content`

Reply:
> actually it looks like disable_until_ts and collab_metadata don't get used in storage.bank_add_content

Might be needed to complete the issue then!